README.md

```diff
@@ -4,118 +4,85 @@
+*Note that Sonobuoy Scanner runs conformance tests only.*
 
 ![tarball overview screenshot][20]
 
-
-## Quickstart
-
-> Heptio provides prebuilt Sonobuoy container images in its Google Container Registry (*gcr.io/heptio-images*). For the sake of faster setup on your cluster, **this quickstart pulls from this registry to skip the container build process**. You can use this same process to deploy Sonobuoy to production.
-
+## Using the CLI
 
 This guide executes a Sonobuoy run on your cluster, and records the following results:
 * Basic info about your cluster's hosts, Kubernetes resources, and versions.
 * *(Via plugin)* [`systemd`][14] logs from each host
-* *(Via plugin)* The results of a single e2e conformance test ("Pods should be submitted and removed"). See the [conformance guide][13] for configuration details.
+* *(Via plugin)* The results of a single e2e conformance test ("Pods should be submitted and removed"). See the [conformance guide][4] for configuration details.
 
 ### 1. Download
-Clone or fork the Sonobuoy repo:
+
+Sonobuoy is written in golang and can easily be obtained by running:
 ```
-git clone https://github.com/heptio/sonobuoy.git
+$ go get github.com/heptio/sonobuoy
 ```
 
 ### 2. Run
 
-First, make sure that you're in your Sonobuoy root directory.
-
 Now you're ready to deploy a Sonobuoy pod to your cluster! Run the following command:
 ```
-kubectl apply -f examples/quickstart.yaml
+$ sonobuoy run
 ```
 
 You can view actively running pods with the following command:
 ```
-kubectl get pods -l component=sonobuoy --namespace=heptio-sonobuoy
+$ sonobuoy status 
 ```
 
-To verify that Sonobuoy has completed successfully, check the logs:
+To inspect the logs:
 ```
-kubectl logs -f sonobuoy --namespace=heptio-sonobuoy
+$ sonobuoy logs
 ```
-If you see the log line `no-exit was specified, sonobuoy is now blocking`, the Sonobuoy pod has completed its data collection.
-
-> *Notes*:
->
-> * **The Sonobuoy pod in this example continues to run after it finishes data collection**, due to the `--no-exit` flag in its YAML manifest. This allows you to easily grab the results tarball.
->
-> * **Sonobuoy collects one data report per run**---each time you want a new report you need to delete the existing Sonobuoy YAML files and reapply them (see the [tear down step][15]).
->
-> * **In practice, you should make sure that the Sonobuoy pod writes its results to a Persistent Volume.** The quickstart example writes its output to an `emptyDir` volume for simplicity.
->
-> *Troubleshooting errors from `kubectl logs`*:
->  * If you are able to debug and resolve the issue on your own, *make sure to delete and reapply Sonobuoy's YAML manifests*. Otherwise, [file an issue][10].
->
 
 To view the output, copy the output directory from the main Sonobuoy pod to somewhere local:
 ```
-kubectl cp heptio-sonobuoy/sonobuoy:/tmp/sonobuoy ./archive --namespace=heptio-sonobuoy
+$ sonobuoy cp .
```

Comment:
supremely minor nit, this is the default location for the cp command so a `$ sonobuoy cp` would do the same